backend/api/groups.py

```diff
@@ -27,11 +42,13 @@ async def create_group(
         request.description (str): The description of the group.
     ```
     """
+    join_code = generate_unique_join_code()
```

Comment:
In the config_info file, a constant named JOIN_CODE_LENGTH should be defined and passed as parameter for this function. Also, i propose no default value for the functions responsible for join code generation